.github/workflows/manual-build.yml

```
@@ -0,0 +1,69 @@
+# Deploy the app to stable. Not builds for development/QA
+name: Manually build latest stable 
+on: 
+  workflow_dispatch: # manually run the action
+
+env:
+  XCODE_VERSION: "14.0"
+  GITHUB_CONTEXT: ${{ toJSON(github) }} # used by fastlane for deploying app 
+  FASTLANE_XCODEBUILD_SETTINGS_RETRIES: 6 # increase timeout for xcodebuild settings. We have seen this command take a bit longer to do on CI server. 
+
+jobs:
+  deploy-app:
+    name: Re-release the latest stable build 
+    runs-on: macos-12
```

```suggestion
  deploy-android-app:
    runs-on: ubuntu-latest
```

We can make the Android and iOS builds run in parallel to speed up builds.

---

I tried `ubuntu-latest` and it gives you errors, initially you have to add support of `x86_64-linux` in Gemlock, then it would give you an error of `pod` command not available etc. 

I could resolve them but the more changes I do the more we are going to deviate from other sample app workflows since all of them currently use `mac-os`. So I am keeping it the same and we can update it later when we move sample apps into the CI workflow.